dev_guide/builds.adoc

```diff
@@ -1104,6 +1214,51 @@ this case `*scmsecret*`:
     }
   }
 ----
-<1> The URL of private repository is usually in the form
-`git@example.com:<username>/<repository>`.
+<1> The URL of private repository, accessed by a private SSH key, is usually
+in the form `git@example.com:<username>/<repository>.git`.
+====
+
+[#other-authentication]
+=== Other
+
+In case the cloning of your application is dependent on a `CA certificate`, `.gitconfig`
+file or both, you can create a secret that contains them and then add it to the builder
+service account.
+
+. Create desider type of `*secret*`:
+
+.. To create a secret from a `.gitconfig`:
++
+====
+----
+$ oc secrets new mysecret .gitconfig=path/to/.gitconfig
+----
+====
+.. To create a secret from a `CA certificate`:
++
+====
+----
+$ oc secrets new mysecret ca.crt=path/to/certificate
+----
+====
+.. To create a secret from a `CA certificate` and `.gitconfig`:
++
+====
+----
+$ oc secrets new mysecret ca.crt=path/to/certificate .gitconfig=path/to/.gitconfig
+----
+====
+
+[NOTE]
+====
+Please note that the SSL verification can be turned off, if `sslVerify=false` is set
+in your `.gitconfig` file.
+====
+
+. Add the `*secret*` to the builder service account:
```

Comment:
You also need to set it in your BuildConfig, but since you have a couple of examples above, I would leave this part out since it's the same as above.